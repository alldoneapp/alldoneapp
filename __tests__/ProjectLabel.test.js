import React from 'react'
import ProjectLabel from '../components/ProjectLabel'

import renderer from 'react-test-renderer'

describe('ProjectLabel component', () => {
    describe('ProjectLabel snapshot test', () => {
        it('should render correctly', () => {
            const tree = renderer.create(<ProjectLabel />).toJSON()
            expect(tree).toMatchSnapshot()
        })
    })
})
