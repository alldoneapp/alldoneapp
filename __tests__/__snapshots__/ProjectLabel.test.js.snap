// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectLabel component ProjectLabel snapshot test should render correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "borderBottomColor": "#E7ECEF",
        "borderBottomWidth": 1,
        "flex": 1,
        "flexDirection": "row",
        "height": 56,
        "paddingBottom": 6,
        "paddingTop": 25,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "borderRadius": 100,
              "height": 16,
              "marginLeft": 4,
              "width": 16,
            },
            Object {
              "backgroundColor": undefined,
            },
          ]
        }
      />
      <Text
        style={
          Object {
            "color": "#04142F",
            "fontFamily": "Roboto-Medium",
            "fontSize": 16,
            "letterSpacing": 0.16,
            "lineHeight": 24,
            "paddingLeft": 8,
          }
        }
      />
    </View>
  </View>
</View>
`;
